data/emoticons.csv

```diff
@@ -0,0 +1,19 @@
+Emote Code,Meaning,Category
```

Comment:
I like that we have descriptions for those who aren't into Twitch culture (such as myself).

Reply:
Haha yes I agree. Its only 17 of the ones that Twitch deemed worthy tho. One dataset used 500 unique emoticons!